client-next/src/util/getApiUrl.ts

```diff
@@ -0,0 +1,12 @@
+const endpoint = import.meta.env.VITE_API_URL;
+
+export const getApiUrl = () => {
+  try {
+    // the URL constructor will throw exception if endpoint is not a valid URL,
+    // e.g. if it is a relative path
+    new URL(endpoint);
```

Comment:
Does this line do anything?

Reply:
Yes, it throws an exception if endpoint is not a valid URL